tests/snapshots.test.js

```diff
@@ -0,0 +1,76 @@
+const {URL} = require('url');
+const snapshotsTest = require('../snapshots');
+
+const PORT = process.env.PORT || 8101;
+
+test('Number of snapshots', async () => {
+  const snapshots = await snapshotsTest();
+  const totalSnapshots = snapshots.reduce((acc, snapshot) => acc + snapshot.widths.length, 0);
+  expect(totalSnapshots).toBe(2906);
+});
```

Comment:
This test would need to be updated every time the total number of snapshots taken by Percy changes, so we may find ourselves updating it frequently. I'll probably remove this but just noting it for the rest of the team.